packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -432,7 +454,7 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
     const { highlightedIndex, open, searchQuery, value } = this.state
 
     return (
-      <ElementType className={classes.root} {...unhandledProps}>
+      <ElementType className={classes.root} onChange={this.handleChange} {...unhandledProps}>
```

Comment:
Aren't we going to end up here with `onChange` handler on a `div` element?

Reply:
I did the same ting as in RadioGroupItem.

```
    // RadioGroupItem component doesn't present any `input` component in markup, however all of our
    // components should handle events transparently.
```